### PR TITLE
Make /getareas work like its Tsu counterpart

### LINKED ISSUE #235: [Feature Request] /getareas shouldn't mention areas that have 0 users in them.
image
Typically, /getareas is used to see who's in every area that's populated, since you could view that an area is empty from the built-in area list on client or web. Reporting all the empty areas makes it harder to sift through, whereas if it didn't do that, you'd only get the users from areas that are in use, which is what you want when using this command.

### PR DESCRIPTION
**Pull request by Salanto**
This is honestly the better variant. ngl.
Fixes #235

### CODE REVIEW

File: core/src/commands/area.cpp [hunk 1]
```diff
@@ -215,10 +215,12 @@ void AOClient::cmdGetAreas(int argc, QStringList argv)
     Q_UNUSED(argv);
 
     QStringList l_entries;
-    l_entries.append("== Area List ==");
+    l_entries.append("\n== Currently Online: " + QString::number(server->m_player_count));
```
Comment (in1tiate) [inline core/src/commands/area.cpp:218]:
```suggestion
    l_entries.append("\n== Currently Online: " + QString::number(server->m_player_count)+ " ==");
```
gotta keep that symmetry

**Review by in1tiate — APPROVED** [review]
looks good

**Comment by Salanto** [comment]
image

**Review by stonedDiscord — COMMENTED** [review]
why do i have to merge this

**Comment by stonedDiscord** [comment]
salanto bullied me to do it

**Comment by Salanto** [comment]
Incorrect.

**Pull request merged**